pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -1,84 +1,5 @@
-%script{src: "/shared/js/angular-resource/angular-resource.min.js"}
-%script{src: "/shared/js/angular-sanitize/angular-sanitize.min.js"}
-%script{src: "/shared/js/ng-csv/ng-csv.min.js"}
-
-%link{rel: 'stylesheet', type: 'text/css', href: '/css/learn-tabs.css'}
-%link{rel: 'stylesheet', type: 'text/css', href: '/style.css'}
-%link{rel: 'stylesheet', type: 'text/css', href: '/css/page.css'}
-%link{rel: 'stylesheet', type: 'text/css', href: '/shared/css/course-blocks.css'}
-%link{rel: 'stylesheet', type: 'text/css', href: '/shared/css/teacher-announcement.css'}
-%link{rel: 'stylesheet', type: 'text/css', href: '/shared/css/teacher-dashboard.css'}
-%link{rel: 'stylesheet', type: 'text/css', href: '/shared/css/user-progress.css'}
-
-- js_locale = request.locale.to_s.downcase.tr('-', '_')
-%script{src: asset_path("js/#{js_locale}/common_locale.js")}
-
-- i18n = {}
-- i18n[:hoc_category_name] = I18n.t('hoc_category_name', default: 'hoc')
-- i18n[:dashboard_students_female] = I18n.t('dashboard_students_female')
-- i18n[:dashboard_students_male] = I18n.t('dashboard_students_male')
-- i18n[:dashboard_submission_submitted] = I18n.t('dashboard_submission_submitted')
-- i18n[:dashboard_submission_in_progress] = I18n.t('dashboard_submission_in_progress')
-- i18n[:dashboard_correctness_free_response] = I18n.t('dashboard_correctness_free_response')
-- i18n[:dashboard_correctness_unsubmitted] = I18n.t('dashboard_correctness_unsubmitted')
-- i18n[:dashboard_correctness_correct] = I18n.t('dashboard_correctness_correct')
-- i18n[:dashboard_correctness_incorrect] = I18n.t('dashboard_correctness_incorrect')
-- i18n[:error_string_none_selected] = I18n.t('dashboard_error_none_selected').gsub(/"/,"\\\\\"")
-- i18n[:error_string_other_section] = I18n.t('dashboard_error_other_section').gsub(/"/,"\\\\\"")
-
+-# This is the entry-point for the old teacher dashboard served from pegasus.
+-# It now redirects all users to the new teacher dashboard served from dashboard.
 - data = {}
-- data[:studiourlprefix] = CDO.studio_url('', CDO.default_scheme)
-- data[:i18n] = i18n
-- data[:valid_login_types] = DashboardSection::valid_login_types
-- data[:valid_grades] = DashboardSection::valid_grades
-- data[:disabled_scripts] = DashboardSection::progress_disabled_scripts(request.user_id)
-
-:ruby
-  gdpr_data = {}
-  user_properties = JSON.parse(dashboard_user[:properties])
-  already_accepted = !user_properties["data_transfer_agreement_accepted"].nil?
-  force_eu = !request.params['force_in_eu'].nil?
-  gdpr_data[:show_gdpr_dialog] = (request.gdpr? || force_eu) && !already_accepted
-  gdpr_data[:current_user_id] = request.user_id
-  data[:gdpr] = gdpr_data
-
-#gdpr-dialog
-
-/We Intentionally take the non-minified JS file here, as it contains a bunch of
-/angular code that runs into problems when minified. Ideally we will fix those
-/issues and minify this code.
-%script{src: asset_path('js/code.org/public/teacher-dashboard/index.js'), data: {teacherdashboard: data.to_json}}
```

Comment:
if you resurrect this file, you should now be able to use `minifiable_asset_path` on it.